### PR TITLE
Move user service

### PR DESCRIPTION
**Pull request by forestik**

### CODE REVIEW

**Review by Markiyan000 — APPROVED** [review]

**Review by olenapetryshak — APPROVED** [review]

File: core/src/main/java/greencity/security/service/impl/FacebookSecurityServiceImpl.java [hunk 1]
```diff
@@ -85,15 +91,15 @@ public SuccessSignInDto generateFacebookAccessToken(String code) {
             .getAccessToken();
         if (accessToken != null) {
             Facebook facebook = new FacebookTemplate(accessToken);
-            User byEmail = facebook.fetchObject("me", User.class, "name", "email");
+            UserVO byEmail = facebook.fetchObject("me", UserVO.class, "name", "email");
```
Comment (vterlyha) [inline core/src/main/java/greencity/security/service/impl/FacebookSecurityServiceImpl.java:94]: Move "me" to constant

File: service-api/src/main/java/greencity/dto/place/PlaceAddDto.java [hunk 2]
```diff
@@ -1,23 +1,18 @@
...
 import javax.validation.Valid;
 import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.Size;
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
+import lombok.*;
```
Comment (vterlyha) [inline service-api/src/main/java/greencity/dto/place/PlaceAddDto.java:15]: Customize your IDEA not to grab imports to *

File: service-api/src/main/java/greencity/dto/socialnetwork/SocialNetworkVO.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+@AllArgsConstructor
+@Getter
+@Setter
+@Builder
+public class SocialNetworkVO {
+    private Long id;
+
+    @Size(min = 1, max = 500)
+    String url;
+    //    SocialNetworkImageVO socialNetworkImage;
```
Comment (vterlyha) [inline service-api/src/main/java/greencity/dto/socialnetwork/SocialNetworkVO.java:18]: Bad approach to comment fields and code

Comment (vterlyha) [reply service-api/src/main/java/greencity/dto/socialnetwork/SocialNetworkVO.java:18]: Just remove it if it is not needed

File: service-api/src/main/java/greencity/dto/user/UserVO.java [hunk 4]
```diff
@@ -16,4 +33,60 @@
     private String email;
 
     private ROLE role;
+
```
Comment (vterlyha) [inline service-api/src/main/java/greencity/dto/user/UserVO.java:36]: Class looks really huge - can it be splitted ?

File: core/src/main/java/greencity/security/service/impl/FacebookSecurityServiceImpl.java (hunk 1 above)
Comment (forestik) [reply core/src/main/java/greencity/security/service/impl/FacebookSecurityServiceImpl.java:94]: Only "me"?

**Review by Yura09 — APPROVED** [review]

File: service-api/src/main/java/greencity/dto/socialnetwork/SocialNetworkVO.java (hunk 3 above)
Comment (forestik) [reply service-api/src/main/java/greencity/dto/socialnetwork/SocialNetworkVO.java:18]: I just tested and forgot about it

**Pull request merged**